client/extensions/woocommerce/app/settings/payments/payment-method-edit.js

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
Thank makes sense but I think is out of the scope of this commit to generalize it.  I'd like to see a use case before doing that as well.